src/framework/components/app/core.clj

```diff
@@ -91,5 +95,5 @@
                (get :response))))))
 
 (defn make-app
-  [config router-interceptors controller-interceptors]
-  (map->App {:config config :router-interceptors router-interceptors :controller-interceptors controller-interceptors}))
+  [config acl-cfg router-interceptors controller-interceptors]
+  (map->App {:config config :acl-cfg acl-cfg :router-interceptors router-interceptors :controller-interceptors controller-interceptors}))
```

Comment:
you do not need to pass `acl-cfg` if it is `(select-keys config [:acl/permissions :acl/roles])`